src/main/java/teammates/logic/api/EmailGenerator.java

```diff
@@ -658,4 +670,30 @@ private EmailWrapper getEmptyEmailAddressedToEmail(String recipient) {
         return email;
     }
 
+    public List<InstructorAttributes> generateCoOwnersList(String courseId) {
```

Comment:
This method seems more suited to be in InstructorsLogic. Maybe you could move this there and rename the method accordingly?